src/libs/TransactionUtils.ts

```diff
@@ -58,16 +58,13 @@ function buildOptimisticTransaction(
         commentJSON.originalTransactionID = originalTransactionID;
     }
 
-    // For the SmartScan to run successfully, we need to pass the merchant field empty to the API
```

Comment:
Can you test all types of money requests?
As comment says, it might break scan request flow

Reply:
@situchan 

Scan request works fine. What I fixed is an optimistic data. API is still called with an empty merchant.

Reply:
@s-alves10 this was added in #25924
Let's again make sure that this doesn't break anything

Reply:
Confirmed working fine. And it's just optimistic data change